tools/find-inactive-collaborators.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
```

Comment:
nit
```suggestion
#!/usr/bin/env node
'use strict';
```

Reply:
If you do this, the file should have the executable bit